3-JSS/src/webpackTodoList.js

```diff
@@ -0,0 +1,148 @@
+import {
+    loadTodoListFromLS,
+    updateTodoInLS,
+    removeTodoFromLocalStorage,
+    saveTodoToLocalStorage,
+    getTodoById} from './localStorage'
+import {classes} from './stylesJSS'
+
+(function() {
```

Comment:
why do u need the iffy?

Reply:
so i talked to Tom about it, and if I wont use iffy, I'll reveal the functions (it will be in the dom). If ill try to access one of the function ill be able to. 
Because im using webpack, the webpack will create this iffy its self so it will do the job for me. I think leaving the iffy makes sense.

Reply:
webpack will add it? did u try?

Reply:
yes if we'll check at the dist directory in the "main" file (which is the file of the app) we will see that it creates an iffy at the end.